_data/3.11-manual.yml

```diff
@@ -644,6 +441,212 @@
       children:
         - text: Agency Dump
           href: troubleshooting-cluster-agency-dump.html
+- subtitle: TOOLS
+- text: Programs & Tools
```

Comment:
Another section with only one chapter in it, and it's not just tools. Would calling it "ecosystem" or something broader make sense? Or split it into Server and Tools?

Reply:
Yes, let's split it into Server and Tools. "Ecosystem" sounds good as a subtitle.